modules/efk-logging-deploy-subscription.adoc

```diff
@@ -55,11 +60,49 @@ metadata:
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
 
-.. Create the Namespace object:
+.. Run the following command to create the project:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
 +
 ----
 $ oc create -f eo-project.yaml
 ----
+  
+.. Create a Namespace for the Cluster Logging Operator (for example, `clo-project.yaml`):
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: "" <1>
+  labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true"
+----
+<1> Optionally specify an empty node selector in order for the logging pods to spread
+evenly across your cluster. The logging pods will be spread evenly throughout the cluster.
+If you want the logging pods to run on specific nodes, you can specify a node selector value here.
+
+.. Run the following command to create the project:
```

Comment:
"project" should be "namespace"

Reply:
Clould we also not do this with a series of oc commands like:
```
oc create ns openshift-logging
oc label ns openshift-logging openshift.io/cluster-logging="true" openshift.io/cluster-monitoring="true"
```

Reply:
sounds plausible

Reply:
@jcantrill I believe that we need `openshift.io/node-selector: ""` in the CLO namespace, which is not created by `oc create`. Can we inject that parameter via the CLI?